### ISSUE TITLE
Run NPM and Java tests in Helix

### ISSUE THREAD

**Issue by anurse** [dotnet/aspnetcore#18799]
We should ensure we are running our SignalR NPM and Java tests in Helix to get full coverage. As part of this, we can review how they appear in the TestResults table to make sure we know how to categorize them properly.

**Comment by BrennanConroy** [comment]
Blocked by https://github.com/dotnet/arcade/issues/4762

**Comment by BrennanConroy** [comment]
NPM tests are on hold for now as they are complex and will likely take more resources than we'd like to spend on them right now.

**Comment by BrennanConroy** [comment]
@anurse should we backlog this for npm?

**Comment by anurse** [comment]
Yeah.
